### ISSUE TITLE
Failure to launch—encountered a 0-size chunk on MS Windows 10

### ISSUE THREAD

**Issue by RT12Productions** [OpenRCT2/OpenRCT2#6263]
OS:  MS Windows 10
Version:  0.1.2
Commit/Build:  2e8d1b5

_OpenRCT2_ hasn't been able to fully launch onto lately.  I've been getting this error message:

**OpenRCT2 V. 0.1.2-develop build 2e8d1b5
encountered a zero-size chunk!**

followed by:

**OpenRCT2 V. 0.1.2-develop build 2e8d1b5
by AppPureyor
failure to reallocate b1f6d040 (and 4206040)
(unsigned char) to have 0 bytes!**

**ABORT, RETRY, or IGNORE**

This has thrown me off from _OpenRCT2_.  What should I be doing to permit me to launch onto the game again?!?!  Thank You!

**Comment by Gymnasiast** [comment]
Do you still have this issue with the newest `develop` version?
If so, when do you get this error? When you're trying to load a park? If so, can you upload that save?

**Comment by RT12Productions** [comment]
Yes!  I'm still getting it with this newest update.

**OpenRCT2 V. 0.1.2-develop build 57be3b4
provided by AppVeyor
Failed to reallocate def3e040 (unsigned char) to have 0 bytes**

It has still encountered a zero-size chunk... .

**Comment by AaronVanGeffen** [comment]
How odd. Many of our developers are on Windows, so these are some rather unique error messages you're getting. What version of Windows are you on, exactly? (32-bits, 64-bits?)

**Comment by janisozaur** [comment]
@RT12Productions can you post your `config.ini` file? Along with that, can you locate your title sequence and if it is anything different than our official one, provide that as well?

**Comment by RT12Productions** [comment]
I don't have a **config.ini** nor any title sequence files.  What should I now do?!?!

**Comment by RT12Productions** [comment]
[config.zip](https://github.com/OpenRCT2/OpenRCT2/files/1765255/config.zip)

**Comment by RT12Productions** [comment]
[general]
always_show_gridlines = false
autosave = 1
confirmation_prompt = true
currency_format = USD
custom_currency_rate = 10
custom_currency_affix = SUFFIX
custom_currency_symbol = "Ctm"
edge_scrolling = true
edge_scrolling_speed = 12
fullscreen_mode = 0
fullscreen_height = 768
fullscreen_width = 1366
rct1_path = ""
game_path = "C:/Program Files/Atari/RollerCoaster Tycoon 2 Triple Thrill Pack"
landscape_smoothing = true
language = en-US
measurement_format = IMPERIAL
play_intro = true
save_plugin_data = true
debugging_tools = true
show_height_as_units = false
temperature_format = FAHRENHEIT
window_height = 600
window_snap_proximity = 5
window_width = 1252
default_display = 0
drawing_engine = SOFTWARE
uncap_fps = false
use_vsync = true
test_unfinished_tracks = true
no_test_crashes = true
date_format = MM/DD/YY
auto_staff = true
handymen_mow_default = true
default_inspection_interval = 2
last_run_version = "0.1.3"
invert_viewport_drag = false
load_save_sort = 0
minimize_fullscreen_focus_loss = true
day_night_cycle = true
enable_light_fx = false
upper_case_banners = false
disable_lightning_effect = false
allow_loading_with_incorrect_checksum = true
steam_overlay_pause = true
window_scale = 1.000000
scale_quality = SMOOTH_NEAREST_NEIGHBOUR
show_fps = false
trap_cursor = false
auto_open_shops = true
scenario_select_mode = 1
scenario_unlocking_enabled = false
scenario_hide_mega_park = false
last_game_directory = "C:\\Users\\RobTe\\OneDrive\\Documents\\OpenRCT2\\save\\"
last_landscape_directory = "C:\\Users\\RobTe\\OneDrive\\Documents\\OpenRCT2\\scenario\\"
last_scenario_directory = "C:\\Users\\RobTe\\OneDrive\\Documents\\OpenRCT2\\scenario\\"
last_track_directory = ""
window_limit = 64
zoom_to_cursor = true
render_weather_effects = true
render_weather_gloom = true
show_guest_purchases = true
show_real_names_of_guests = true
use_virtual_floor = true
allow_early_completion = false

[interface]
toolbar_show_finances = true
toolbar_show_research = true
toolbar_show_cheats = true
toolbar_show_news = true
toolbar_show_mute = true
console_small_font = false
current_theme = "*RCT2"
current_title_sequence = "*OPENRCT2"
object_selection_filter_flags = 8463

[sound]
master_volume = 100
title_music = 2
sound = true
sound_volume = 100
ride_music = true
ride_music_volume = 100
audio_focus = false
audio_device = ""

[network]
player_name = "RobTele1412"
default_port = 11753
listen_address = ""
default_password = ""
stay_connected = true
advertise = true
maxplayers = 16
server_name = "Server"
server_description = ""
server_greeting = ""
master_server_url = ""
provider_name = ""
provider_email = ""
provider_website = ""
known_keys_only = false
log_chat = false
log_server_actions = false
pause_server_if_no_clients = false

[notifications]
park_award = true
park_marketing_campaign_finished = true
park_warnings = true
park_rating_warnings = true
ride_broken_down = true
ride_crashed = true
ride_warnings = true
ride_researched = true
guest_warnings = true
guest_lost = false
guest_left_park = true
guest_queuing_for_ride = true
guest_on_ride = true
guest_left_ride = true
guest_bought_item = true
guest_used_facility = true
guest_died = true

[twitch]
channel = ""
follower_peep_names = true
follower_peep_tracking = true
chat_peep_names = true
chat_peep_tracking = true
news = false

[font]
file_name = ""
font_name = ""
x_offset = 0
y_offset = 1
size_tiny = 1
size_small = 0
size_medium = 0
size_big = 0
height_tiny = 0
height_small = 0
height_medium = 0
height_big = 0
enable_hinting = true
hinting_threshold = 0

**Comment by janisozaur** [comment]
There's little we can do with that without a proper dump. You can get one easily with procexp.

**Comment by janisozaur** [comment]
OpenRCT2 should also report such failures to `stderr`, you can check the name of offending file there and send it out way for inspection.

**Comment by Gymnasiast** [comment]
In layman's terms, it means that it will print output if you run openrct2 from a Command Prompt window.

**Comment by RT12Productions** [comment]
It turns out to be the Launcher that has been giving me these errors...
I’ve since started *OpenRCT2* by its own desktop icon (that doesn’t involve
the Launcher), and now works fine.  Thank You!


On Tue, Apr 10, 2018 at 3:37 AM, Michael Steenbeek <notifications@github.com
